packages/optimistic-oracle/index.js

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+require("dotenv").config();
+const retry = require("async-retry");
+
+const { Logger, waitForLogger, delay } = require("@uma/financial-templates-lib");
+
+// Contract ABIs and network Addresses.
+const { getAbi, getAddress } = require("@uma/core");
+const { getWeb3 } = require("@uma/common");
+
+/**
+ * @notice Runs strategies that propose and dispute prices for any price identifier serviced by the Optimistic Oracle.
+ * @param {Object} logger Module responsible for sending logs.
+ * @param {Object} web3 web3.js instance with unlocked wallets used for all on-chain connections.
+ * @param {Number} pollingDelay The amount of seconds to wait between iterations. If set to 0 then running in serverless
+ *     mode which will exit after the loop.
+ * @param {Number} errorRetries The number of times the execution loop will re-try before throwing if an error occurs.
+ * @param {Number} errorRetriesTimeout The amount of milliseconds to wait between re-try iterations on failed loops.
+ * @return None or throws an Error.
+ */
+async function run({ logger, web3, pollingDelay, errorRetries, errorRetriesTimeout }) {
+  try {
+    // Load unlocked web3 accounts and get the networkId.
+    const [networkId] = await Promise.all([web3.eth.net.getId()]);
+
+    // Setup contract instances. NOTE that getAddress("OptimisticOracle", networkId) will resolve to null in tests.
+    const oracle = new web3.eth.Contract(getAbi("OptimisticOracle"), getAddress("OptimisticOracle", networkId));
+
+    // If pollingDelay === 0 then the bot is running in serverless mode and should send a `debug` level log.
+    // Else, if running in loop mode (pollingDelay != 0), then it should send a `info` level log.
+    logger[pollingDelay === 0 ? "debug" : "info"]({
+      at: "OptimisticOracle#index",
+      message: "OO keeper started 🌊",
+      oracleAddress: oracle.address,
+      pollingDelay,
+      errorRetries,
+      errorRetriesTimeout
+    });
+
+    // TODO:
+    // - Miscellaneous setup
+    // - Instantiate an OO client
+    // - Run proposer and/or disputer strategy
+
+    // Create a execution loop that will run indefinitely (or yield early if in serverless mode)
+    for (;;) {
+      await retry(
+        async () => {
+          return true;
```

Comment:
What does the Boolean signify/do?

Reply:
It's a placeholder for now I'll make that more clear

Reply:
Sorry, a bit more context: in the other bots, this function doesn't return anything. Any reason to return a Boolean over just `return;`?

Reply:
Good point I'll just return nothing